projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-saved-cart.ts

```diff
@@ -590,50 +593,92 @@ export function updateSavedCartAndRestore(
 
         alerts.getSuccessAlert().should('contain', `Cart Edited Successfully`);
 
-        // restore saved cart
-        const restoreSavedCartAlias = interceptRestoreSavedCartEndpoint(
-          cart.code
-        );
-        const getAllSavedCartAlias = interceptGetAllSavedCartEndpoint();
+        if (addToActiveCart) {
```

Comment:
Can it be broken into two:  AddToActiveCart and RestoreToCart?

Reply:
yes!